pages/common/aws-s3-ls.md

```diff
@@ -0,0 +1,28 @@
+# aws s3 ls
+
+> List AWS S3 buckets, folders (prefixes), and files (objects).
+> More information: <https://awscli.amazonaws.com/v2/documentation/api/latest/reference/s3/ls.html>.
+
+- List all buckets:
+
+`aws s3 ls`
+
+- List files and folders in the root of a bucket (`s3://` is optional):
+
+`aws s3 ls s3://{{bucket_name}}`
+
+- List files and folders directly inside a directory:
+
+`aws s3 ls {{bucket_name}}/{{path/to/directory/}}`
```

Comment:
The `/` is required?
```suggestion
`aws s3 ls {{bucket_name}}/{{path/to/directory}}`
```

Reply:
Yes, it is!
Without it you would see something like:
```
PRE directory/
```

Reply:
Then:
```suggestion
`aws s3 ls {{bucket_name}}/{{path/to/directory}}/`
```

Reply:
Ok